Gruntfile.js

```diff
@@ -120,10 +120,31 @@ module.exports = (grunt) => {
         }
     });
 
+    grunt.loadNpmTasks('grunt-webext-builder');
     grunt.initConfig({
         manifest: grunt.file.readJSON('manifest.json'),
-        cwsOptions: grunt.file.readJSON('cws.json'),
         webpack: webpackConfigs,
+        webext_builder: {
+            chrome: {
+                privateKey: ".private.pem",
```

Comment:
I think it's better to use something more descriptive as the filename, eg. `.chrome-ext-sign-private.pem`.